ui/lib/core/addon/components/search-select.js

```diff
@@ -131,11 +131,6 @@ export default Component.extend({
       this.handleChange();
     },
     selectOption(option) {
-      if (this.selectLimit) {
```

Comment:
Does selectLimit do anything if this is removed?

Reply:
Yes.  selectLimit now prevents the select from ever showing, so the handleChange never comes into play.  It goes from model to template, no need for it to hit the component js file.

Reply:
It's hard to see in this PR, because it was done in a previous one, but the template `search-select.hbs` has a line on row 22: `{{#unless (gte selectedOptions.length selectLimit)}}...` that provides the conditional.